panolint-rubocop.yml

```diff
@@ -180,39 +183,3 @@ RSpec/ExampleLength:
 
 RSpec/MultipleExpectations:
   Enabled: false
-
-################################################################################
-# Because of rubocop's new versioning scheme, you get a warning if you don't
-# have config set for specific cops until the next major version comes out. The
-# following rules all mirror the default rubocop configuration, but they're here
-# just to silence the warnings. They can be removed when 1.0 comes out.
-
-Lint/DeprecatedOpenSSLConstant:
-  Enabled: true
-
-Layout/EmptyLinesAroundAttributeAccessor:
-  Enabled: true
-
-Layout/SpaceAroundMethodCallOperator:
-  Enabled: true
-
-Lint/RaiseException:
-  Enabled: true
-
-Lint/StructNewOverride:
-  Enabled: true
-
-Style/ExponentialNotation:
-  Enabled: true
-
-Style/HashEachMethods:
-  Enabled: true
-
-Style/HashTransformKeys:
-  Enabled: true
-
-Style/HashTransformValues:
-  Enabled: true
-
-Style/SlicingWithRange:
-  Enabled: true
```

Comment:
I confirmed that all of these cops are enabled by default now, so I don't think we'll get any more of those warnings. Rubocop now has a default setting we use above for new cops (l. 22) and it looks like 1.0.0 came out in 2020.